.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
.editorconfig files are not a very common thing, but I'm curious if others think it would be helpful

Reply:
I don't have an opinion here.  @beurdouche @raphaelrobert ?